lib/sles4sap.pm

```diff
@@ -122,9 +122,10 @@ sub test_start_service {
     my $output = script_output "sapcontrol -nr $instance -function StartService $sid";
     die "sapcontrol: StartService API failed\n\n$output" unless ($output =~ /StartService[\r\n]+OK/);
 
-    $output = script_output $ps_cmd;
+    # We can't use the $ps_cmd alias, as number of process can be >1 on some HANA version
+    $output = script_output "ps auxww | grep '$sid.*sapstartsrv | grep -vw grep'";
```

Comment:
There is something off with the quotes within the double quotes there. IMO, it should be:

```
script_output "ps auxww | grep '$sid.*sapstartsrv' | grep -vw grep";
```

Even the verification run is showing a false positive: http://1b147.qa.suse.de/tests/5360#step/hana_test/48

Reply:
Fixed, should be better with `pgrep` (as suggested by @ricardobranco777). Will see with the current run.